### DISCUSSION TITLE
Sustainability - EKG/Principles

### DISCUSSION THREAD

**Discussion by giscus** [EKGF/ekg-principles#23] [Page Comments]
# Sustainability - EKG/Principles

Manifesto and principles for the Enterprise Knowledge Graph (EKG)

https://principles.ekgf.org/vocab/sustainability/

**Comment by jgeluk** [comment]
Make the link to scalability here

**Reply by jgeluk** [reply]
"Your method of scalability is not sustainable"

**Comment by ctubbax** [comment]
Values & Ideals
1.	Transparency, openness, sustainability, etc are no longer just terms in the vocab section but themes. So, the hyperlinks of these themes should refer to their theme pages.
2.	I also think human capital is important but it is not clear why human capital is important in the context of EKG. Without that explanation, it looks like a generic statement without further explanation.
3.	Same as last comment
4.	No comment
5.	No comment
6.	Information bubbles fall within silos as a broader term. I think we could better refer to silo and write a great article about it.
7.	A hyperlink referring to the power of abstraction and traceability themes could be added because that bullet point nicely fits with that theme.
8.	 Same as last comment but this should refer to the reusability and composable business themes.

All data connected
1.	No comment
2.	No comment
3.	This could refer to the sustainability theme.
4.	No comment
5.	No comment
6.	I think it should be nice to mention that connecting data in a scalable and sustainable way are a key priority for EKG.

Standards
1.	Standards allow you to not having to reinvent the wheel all the time which is key for scalability and interoperability.
2.	Some examples of such standard should be given to make this bullet point clearer.
3.	Why are machine-readable models interesting from a business point of view? That part is not clear.

Data markets
1.	No comment
2.	No comment
3.	No comment
4.	No comment
5.	No comment
6.	A link with the digital assets theme should be added.

Open world
1.	No comment
2.	No comment
3.	No comment
4.	No comment
5.	No comment
6.	No comment
7.	No comment
8.	No comment
